packages/spartan-council/test/integration/Cast.test.js

```diff
@@ -99,7 +99,9 @@ describe('ElectionModule (cast)', function () {
     const votingPeriodStartDate = epochEndDate - daysToSeconds(7);
     const nominationPeriodStartDate = votingPeriodStartDate - daysToSeconds(7);
 
-    await ElectionModule.initializeElectionModule(
+    await ElectionModule[
+      'initializeElectionModule(string,string,address[],uint8,uint64,uint64,uint64,address)'
+    ](
```

Comment:
This is kinda ugly, but explicit. The SNX specific ElectionModule now has two initialize functions in an overload. This is how you tell ethers which one you want to call. We want to call the one that has an additional parameter that specifies the v2x debt share contract address.